twisted/spread/jelly.py

```diff
@@ -68,9 +68,15 @@
 import decimal
 from functools import reduce
 import copy
-
 import datetime
 
+try:
+    from types import (ClassType as _OldStyleClass,
+                       InstanceType as _OldStyleInstance)
+except ImportError:
+    _OldStyleClass = None
+    _OldStyleInstance = None
```

Comment:
For the same reason as [this comment](https://github.com/twisted/twisted/pull/381/files#r71810390):
`_OldStyleInstance = ()`

I think this stuff probably deserves a comment.